package.json

```diff
@@ -1,8 +1,11 @@
 {
   "name": "@launchpadlab/prettier-config",
-  "version": "1.0.0",
+  "version": "2.0.0",
   "main": "index.json",
   "repository": "https://github.com/launchpadlab/prettier-config",
   "author": "dpikt",
-  "license": "MIT"
+  "license": "MIT",
+  "peerDependencies": {
+    "prettier": "^1.0 || ^2.0 || ^3.0"
```

Comment:
This matches our setup in `esling-config`. It also is a slightly safer configuration, now that we're only including LPL-specific overrides. If/when the next major change is released, we can review the options and ensure that no changes need to be made to accommodate v4.